assets/src/dashboard/app/views/myStories/content/storyGridView/index.js

```diff
@@ -94,19 +77,22 @@ const StoryGridView = ({
   useEffect(() => {
     if (
       !activeGridItemId &&
-      returnStoryFocusId.value &&
+      returnStoryFocusId?.value &&
       !activeGridItemIdRef.current
     ) {
       const newFocusId = returnStoryFocusId?.value;
-      itemRefs.current?.[newFocusId]?.children?.[0]?.focus();
+      setActiveGridItemId(newFocusId);
+      // grab the menu button and refocus
+      itemRefs.current?.[newFocusId]
+        ?.querySelector('.context-menu-button')
```

Comment:
💭 I'm guessing this class is also used on the button somewhere else. Maybe we should make `context-menu-button` a constant and share it just in case it is changed in the future?

Reply:
made it a constant :)